packages/react/src/ActionList/Item.tsx

```diff
@@ -259,8 +268,24 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
     const inlineDescriptionId = `${itemId}--inline-description`
     const blockDescriptionId = `${itemId}--block-description`
     const inactiveWarningId = inactive && !showInactiveIndicator ? `${itemId}--warning-message` : undefined
+    const validRole = listRole === 'listbox' || listRole === 'menu' || inactive
+
+    const ButtonItemWrapper = buttonSemantics
+      ? (React.forwardRef(({as: Component = 'button', children, ...props}, forwardedRef) => {
+          return (
+            <Box
```

Comment:
`<Box>` is used here to provide a non-styled `<button>`, this also goes for anything that's passed to it via `as`. We could use a `<Button>` here, but I'm not sure if it'll provide more than the current implementation, but I may be wrong here!

Reply:
I think you are right, because both render as buttons semantically at the end of the day. Unless we need to utilize Button props, I think we are good with the Box!